internal/provider/resource_ably_rule_http.go

```diff
@@ -28,7 +28,8 @@ func (r resourceRuleHTTP) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagn
 				Optional:    true,
 				Description: "The signing key ID for use in batch mode. Ably will optionally sign the payload using an API key ensuring your servers can validate the payload using the private API key. See the [webhook security docs](https://ably.com/docs/general/webhooks#security) for more information",
 			},
-			"format": GetFormatSchema(),
+			"format":    GetFormatSchema(),
+			"enveloped": GetEnvelopedchema(),
```

Comment:
Should this be `GetEnvelopedSchema()`?

Reply:
It's a typo throughout the codebase :D But I'll fix it now!

Reply:
https://github.com/ably/terraform-provider-ably/pull/167/commits/90f4cf033bad2e627a5fb9ab5cb5130ae1e2765b

Reply:
👍